src/Components/QuestionList/QuestionFilter/QuestionFiltersS.scss

```diff
@@ -2,4 +2,9 @@
     display: flex;
     justify-content: flex-end;
     margin-bottom: 5rem;
+
+    .buttonGroup {
+        display: flex;
+        justify-content: space-between;
+    }
```

Comment:
Bootstrap has a button group already, either use that or change this to buttons maybe?